sdk/containerregistry/Azure.Containers.ContainerRegistry/api/Azure.Containers.ContainerRegistry.netstandard2.0.cs

```diff
@@ -110,6 +110,12 @@ public partial class ArtifactTagProperties
         public string RegistryLoginServer { get { throw null; } }
         public string RepositoryName { get { throw null; } }
     }
+    public static partial class AzureContainerRegistryModelFactory
```

Comment:
@AlexanderSher, can you help me understand what changed in the model factory generator that changed the name of this class?  Thanks!